docker/pennylane.dockerfile

```diff
@@ -39,13 +39,14 @@ ENV PATH="/opt/venv/bin:$PATH"
 #Setup and Build pennylane
 WORKDIR /opt/pennylane
 COPY  . .
-RUN git submodule update --init --recursive \
-    && pip install wheel && pip install -r requirements.txt \
-    && python3 setup.py install \
-    && pip install pytest pytest-cov pytest-mock flaky \
-    && pip install -i https://test.pypi.org/simple/ pennylane-lightning --pre --upgrade \
-    && pip install openfermionpyscf \
-    && make test && make coverage
+RUN git submodule update --init --recursive
+RUN pip install wheel && pip install -r requirements.txt
+RUN python3 setup.py install
+RUN pip install pytest pytest-cov pytest-mock flaky
+RUN pip install -i https://test.pypi.org/simple/ pennylane-lightning --pre --upgrade
+# hotfix, remove when pyscf 2.1 is released (currently no wheel for python3.10)
+RUN pip install openfermionpyscf || true
```

Comment:
I don't understand the change where you replace && with run. Does || true mean that if impossible to install it is skipped?

Reply:
Ah, I changed to using `RUN` instead of `&&` for this command because it was required to isolate the `openfermionpyscf` installation step. `RUN` executes a single command it is given, thus chaining all the installations together into a single command with `&&` not only makes it harder to identify which step specifically fails in the logs, but I only wanted to ignore a failure on the `openfermionpyscf` installation step, not on any of the other packages. `||` just computes the or of the given expressions, so when `pip install` runs and returns a non-zero (=false) error code, it is ignored through "or true". The build then continues on because the command inside `RUN` did not return an error code.